core/src/main/scala/org/broadinstitute/dsp/workbench/welder/GoogleStorageInterp.scala

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.dsp.workbench.welder
+
+import java.time.Instant
+import java.util.concurrent.TimeUnit
+
+import _root_.io.chrisdavenport.linebacker.Linebacker
+import _root_.io.chrisdavenport.log4cats.Logger
+import cats.effect.{ContextShift, IO, Timer}
+import cats.implicits._
+import fs2.{Stream, io}
+import org.broadinstitute.dsde.workbench.google2
+import org.broadinstitute.dsde.workbench.google2.{Crc32, GoogleStorageService, RemoveObjectResult}
+import org.broadinstitute.dsde.workbench.model.{TraceId, WorkbenchEmail}
+import org.broadinstitute.dsp.workbench.welder.GoogleStorageAlg._
+import org.broadinstitute.dsp.workbench.welder.SourceUri.GsPath
+import scala.collection.JavaConverters._
+
+class GoogleStorageInterp(config: GoogleStorageAlgConfig , googleStorageService: GoogleStorageService[IO])
+                         (implicit logger: Logger[IO], timer: Timer[IO], linerBacker: Linebacker[IO], cs: ContextShift[IO]) extends GoogleStorageAlg {
+  def updateMetadata(gsPath: GsPath, traceId: TraceId, metadata: Map[String, String]): IO[Unit] = googleStorageService.setObjectMetadata(gsPath.bucketName, gsPath.blobName, metadata, Option(traceId)).compile.drain.handleErrorWith {
```

Comment:
nit: can you bump the first line of this function onto the next line?

the same comment will apply to a few of the others. it helps with readability IMO and is currently inconsistent with other functions in this class

Reply:
sure yea..I'll run scalafmt, it should fix this